state/machine.go

```diff
@@ -507,23 +508,59 @@ func (m *Machine) ForceDestroy() error {
 		return errors.Trace(err)
 	}
 	if err := m.st.db().RunTransaction(ops); err != txn.ErrAborted {
-		return errors.Trace(err)
+		return errors.Annotatef(err, "failed to run transaction: %s", pretty.Sprint(ops))
```

Comment:
I think I saw these in test failures, vs being in logs. Looking at the code path, it looks like its actually returned to the user.
Specifically
apiserver/facades/client/client/client.go DestroyMachines calls common.DestroyMachines builds up any errors returned by callers, and then turns that into an error.
It may be that we don't want to return the error, but just log it and just return the simple error?